features/support/env.rb

```diff
@@ -3,14 +3,28 @@
 require 'open4'
 require 'rspec'
 
+# rubocop:disable all
```

Comment:
Yeah need to clean that up. Was have trouble geting rubocop to ignore line length though.
`# rubocop:disable Metrics/LineLength` didn't work